scripts/ipxe-dts.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+error_exit() {
+    _error_msg="$1"
+    echo "$_error_msg"
+    exit 1
+}
+
+error_check() {
+    _error_code=$?
+    _error_msg="$1"
+    [ "$_error_code" -ne 0 ] && error_exit "$_error_msg : ($_error_code)"
+}
+
+userIP=$(ip route get 1 | sed 's/^.*src \([^ ]*\).*$/\1/;q')
+error_check "cannot get user IP"
+
+distroVersion=$(cat meta-dts/meta-dts-distro/conf/distro/dts-distro.conf | grep -F "DISTRO_VERSION" | awk -F'"' '{print $2}')
+error_check "cannot get distro version from 'meta-dts/meta-dts-distro/conf/distro/dts-distro.conf'"
+
+cd meta-dts/scripts/
+error_check "cannot cd to 'meta-dts/scripts'"
+
+mkdir -p ipxe-files
+error_check "cannot make directory 'ipxe-files'"
+
+cd ipxe-files
+error_check "cannot cd to 'ipxe-files'"
+
+wget --output-document=bzImage-v$distroVersion http://boot.dasharo.com/dts/v$distroVersion/bzImage-v$distroVersion
+error_check "cannot wget the bzImage v'$distroVersion' from 'http://boot.dasharo.com/dts/v$distroVersion/bzImage-v$distroVersion'"
```

Comment:
```suggestion
error_check "cannot wget the bzImage v$distroVersion from https://boot.dasharo.com/dts/v$distroVersion/bzImage-v$distroVersion"
```

Reply:
There is no need to use ' as we are already using double quotations

Reply:
@TomaszAIR adressed [here](https://github.com/Dasharo/meta-dts/pull/96/commits/353a8ac8fbb8f13b2a94ab193ec04725edf7dc30)